spec/in_paralell_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rspec'
 require_relative('../lib/in_parallel')
 include InParallel
-TMP_FILE = '/tmp/test_file.txt'
+TMP_FILE = 'tmp/test_file.txt'
```

Comment:
Hmm, I don't see why that is a restriction to assigning `TMP_FILE =  Tempfile.new.path` before any tests are run. This doesn't seem to be in a parallel block...